scripts/publish_prod.sh

```diff
@@ -37,12 +37,12 @@ fi
 yarn login
 
 echo "Bumping the version number and committing the changes"
-yarn version --new-version "$VERSION"
+yarn bump
```

Comment:
Does this let us specify a version number? Is this necessarily going to match the version number passed into the script?

Reply:
I changed the publish script to keep versions in sync. I am using "standard-version" in favor of "yarn version" because standard-version is configured to only bump the `version.json` file then subsequently running `npx projen` to sync the `package.json` version with the newly bumped version. This means the package.json should be completely maintained by projen.